api/v1beta1/moduleimagesconfig_types.go

```diff
@@ -39,6 +39,9 @@ type ModuleImageSpec struct {
 	// image
 	Image string `json:"image"`
 
+	// kernel version for which this image is targeted
+	KernelVersion string `json:"kernelVersion"`
```

Comment:
If it is needed by the MBSC then lets add it to the MBSC object as well in the same PR as this is the same logical change (and the same commit message).